plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -717,18 +773,25 @@ function MosaicMenuItem:paintTo(bb, x, y)
     end
 
     -- to which we paint over a dogear if needed
-    if corner_mark and self.do_hint_opened and self.been_opened then
-        -- align it on bottom right corner of sub-widget
-        local target =  self[1][1][1]
-        local ix
-        if BD.mirroredUILayout() then
-            ix = math.floor((self.width - target.dimen.w)/2)
-        else
-            ix = self.width - math.ceil((self.width - target.dimen.w)/2) - corner_mark:getSize().w
-        end
-        local iy = self.height - math.ceil((self.height - target.dimen.h)/2) - corner_mark:getSize().h
-        -- math.ceil() makes it looks better than math.floor()
-        corner_mark:paintTo(bb, x+ix, y+iy)
+    -- if corner_mark and self.do_hint_opened and self.been_opened then
+    --     -- align it on bottom right corner of sub-widget
+    --     local target =  self[1][1][1]
+    --     local ix
```

Comment:
Please leave the whole corner_mark alone :) (holding it captive behind comments makes me feel you will end up murdering it and dispose of the body :)
If you don't want it painted while you are testing, just add a `if false and ...` here:
`if false and corner_mark and self.do_hint_opened and self.been_opened then`